NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 17.3.0 IN-PROGRESS
+
+* Updated Vert.x to version 4 (MODUSERS-242)
+
 ## 17.2.0 2020-10-06
```

Comment:
Please do not tamper with NEWS.. You don't know what next release is going to hold, and nor do you know the date of the release. As long as commits are clear. If you assume this issue is going be fixed for 17.3.0, then update the Jira. Thx

Reply:
@adamdickmeiss 

I agree that the date of the release is unknown. I provided this feedback to @K-Felk previously and he corrected it. I think the release version is relatively known, given that it is defined at the previous release and only a compatibility breaking change will affect that (at which point, the news can be updated).

It was agreed by the Technical Leads [last year|https://wiki.folio.org/display/TLG/2020-06-24+Meeting+notes] that it is [acceptable to change|https://wiki.folio.org/display/DD/Pull+Request+Guidelines] the news during a pull request at the discretion of the lead maintainer.

I am ok with (rather I prefer) the news being changed during pull requests, as it reduces the burden on the person making the release and allows the author of the change to describe it (rather than the releasor having to guess). I believe this is common practice for the front end modules.

I will raise a JIRA issue and pull request to make this policy explicit for this module. Apologies for the inconvenience.